search-app/src/cmr/search/services/parameters/converters/spatial.clj

```diff
@@ -37,3 +38,23 @@
 (defmethod p/parameter->condition :line
   [_context concept-type param value options]
   (url-value->spatial-conditions :line value))
+
+(defmethod p/parameter->condition :wkt
+  [_context concept-type param value options]
+  (def keyw (first value))
+  (def p1 (str/trim (re-find #".+?(?=[(])" keyw)))
+  (def p1 (str/lower-case p1))
+  (def v1 (subs (first (re-find #"([(](.+?)[)])" keyw) ) 1) )
```

Comment:
When you want to perform a bunch of transformations it is good to use the thread-first or thread-last macros.
e.g.
```
(let [v1 (-> (subs (first (re-find #"([(](.+?)[)])" keyw) ) 1) )
             (str/replace #"[()]" ""))
             (str/replace #"[,]" " "))
             str/trim  ; Note if a function only takes one parameter you don't need to wrap in parens in the thread first macro
             ...
```